tests/framework/pvc.go

```diff
@@ -225,6 +225,11 @@ func (f *Framework) GetDiskGroup(namespace *k8sv1.Namespace, pvc *k8sv1.Persiste
 	output, err = f.ExecShellInPod(executorPod.Name, namespace.Name, cmd)
 	fmt.Fprintf(ginkgo.GinkgoWriter, "INFO: gid of disk.img: %s\n", string(output))
 
+	if deletePod {
+		err = f.K8sClient.CoreV1().Pods(namespace.Name).Delete(executorPod.Name, nil)
```

Comment:
Only question here, do we want to wait to ensure the pod is gone before continuing? I have seen some instances where we assumed after a delete, the pod was gone and then checked for the pod being gone, and it was still in terminating or something and that failed the test.

Reply:
Yeah that can take awhile though.  And it appeared to me that there is some effort underway to make things (reusing the verifier pods).  Waiting for the pod to be gone may make debugging failed tests easier.  But I don't think it won't keep tests from failing.  And may make things slower.  Your call.

Reply:
Yeah, I am trying to not delete the pod unless its needed, because it may take a while, but in this case we have explicitly said delete this pod, and the rest of the test might assume its actually gone, while it might still be terminating and cause issues. For this particular PR I don't think that any thing assumes it though.